monkey/monkey_island/cc/services/attack/technique_reports/attack_technique_test.py

```diff
@@ -0,0 +1,97 @@
+import json
+from unittest import TestCase
+
+from common.utils.attack_utils import ScanStatus
+from monkey_island.cc.database import mongo
+
+from . import AttackTechnique
+from .pba_technique import PostBreachTechnique
+
+
+class T9999(PostBreachTechnique):
+    tech_id = "T9999"
+    unscanned_msg = "Unscanned"
+    scanned_msg = "Scanned"
+    used_msg = "Used"
+    pba_names = ["PBA Name"]
+
+
+config =\
+    {
+        'category': {
+            'link': '',
+            'properties': {
+                'T9999': {
+                    'description': '',
+                    'link': '',
+                    'necessary': False,
+                    'title': '',
+                    'type': 'bool',
+                    'value': False  # this field denotes whether technique is enabled or disabled in config
+                }
+            }
+        }
+    }
+
+
+def set_config(value: bool):
+    config['category']['properties']['T9999']['value'] = value
+    return config
+
+
+param_list = [
+    # unscanned
+    (
+        {  # telemetry data
+            'info': [],
+            'message': 'Unscanned',
+            'status': 0,
```

Comment:
Change this to the const from the Enum (import it) instead of number (the test will fail if we choose to change the numbers)

Reply:
Done